src/Select.purs

```diff
@@ -162,11 +161,17 @@ data SelectEvent
   | VisibilityChangedTo Visibility
   | SelectedIndex Int
 
-newtype UseSelect hooks = UseSelect
-  (UseDebouncer String
-  (UseState SelectState hooks))
+foreign import data UseSelect :: Hooks.HookType
 
-derive instance newtypeUseSelect :: Newtype (UseSelect hooks) _
+type UseSelect' =
+  UseState SelectState
+    <> UseDebouncer String
+    <> UseEffect
+    <> Hooks.Pure
+
+instance newtypeUseDebouncer
+  :: HookEquals UseSelect' h
+  => HookNewtype UseSelect h
```

Comment:
`hooks v0.5.0` drops `HookEquals` as its already possible to use type synonyms in this context:
```suggestion
  => HookNewtype UseSelect UseSelect'
```